packages/modules/imodel-browser/src/containers/iModelTiles/IModelGhostTile.tsx

```diff
@@ -11,28 +11,32 @@ import React from "react";
 export const IModelGhostTile = () => {
   return (
     <ThemeProvider theme="inherit">
-      <Tile
-        name={
-          <span
-            className={"iui-skeleton"}
-            style={{ width: "180px", margin: 0 }}
-          >
+      <Tile.Wrapper>
+        <Tile.ThumbnailArea>
+          <Text
+            isSkeleton={true}
+            style={{
```

Comment:
None of these seem required except newly added border radius, which is a nice fix.

Reply:
This change was recommended by Raphael https://github.com/iTwin/admin-components-react/pull/107#discussion_r1401153135